### PR TITLE
Live APY on Landing Page

### PR DESCRIPTION
**Pull request by 0knxwledge**
Addresses Issue #41

### CODE REVIEW

File: src/components/Landing/Landing.vue [hunk 1]
```diff
@@ -337,12 +337,16 @@ import ImageVue from "../Handlers/ImageVue";
...
+  SGT_uniswap,
+  geyser_SGT_uniswap,
+} from "@/contracts";
 export default {
   components: { ImageVue },
   props: ["scrolled", "windowWidth"],
   data() {
     return {
-      TVL: 12000,
+      TVL: 12000, APY: "",
```
Comment (sharedStake-dev) [inline src/components/Landing/Landing.vue:349]: lets put this on a newline

File: src/components/Landing/Landing.vue [hunk 2]
```diff
@@ -352,6 +356,26 @@ export default {
         "https://api.etherscan.io/api?module=stats&action=tokensupply&contractaddress=0x898bad2774eb97cf6b94605677f43b41871410b1&apikey=GKKIY3WXXG1EICPRKACRR75MA4UE7ANFY8"
       );
       this.TVL = BN(response.data.result).div(1e18).toFixed(0).toString();
+
+      let token = SGT_uniswap;
```
Comment (sharedStake-dev) [inline src/components/Landing/Landing.vue:360]: Thanks for doing this! 
For the sake of readability/clean code, can we please extract this into a standalone function? 
So we can use it like:   
```
this.APY = getAPY()
```

**Review by sharedStake-dev — CHANGES_REQUESTED** [review]

File: src/components/Landing/Landing.vue (hunk 2 above)
Comment (0knxwledge) [reply src/components/Landing/Landing.vue:370]: created standalone function

**Comment by sharedStake-dev** [comment]
Verified locally - chimera

**Review by sharedStake-dev — APPROVED** [review]

**Comment by sharedStake-main** [comment]
getAPY  works if any wallet is connected.
We need  a solution **without** using web3 here. 
For that:
- Create an api endpoint that will update APY every once in a while.
- Request apy from api endpoint on Mounted()

**Pull request merged**